openstack_controller/tests/test_unit_glance.py

```diff
@@ -334,3 +334,33 @@ def test_images_pagination(
             tags=metric['tags'],
             hostname=metric.get('hostname'),
         )
+
+
+@pytest.mark.parametrize(
+    ('instance'),
+    [
+        pytest.param(
+            configs.REST,
+            id='api rest no microversion',
+        ),
+        pytest.param(
+            configs.SDK,
+            id='api sdk no microversion',
+        ),
+    ],
+)
+@pytest.mark.usefixtures('mock_http_get', 'mock_http_post', 'openstack_connection')
+def test_disable_glance_image_metrics(aggregator, dd_run_check, instance, openstack_controller_check):
+    instance = instance | {
+        "components": {
+            "image": {
+                "images": False,
+            },
+        },
+    }
+    check = openstack_controller_check(instance)
+    dd_run_check(check)
+    for metric in aggregator.metric_names:
+        assert not metric.startswith('openstack.cinder.image.count')
```

Comment:
I think the correct format is `openstack.glance.image`
In any case, I think this test is already implmented (check line 67 in this file)

Reply:
Sounds good, I have removed the redundant test.